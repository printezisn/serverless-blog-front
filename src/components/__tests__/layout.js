import React from "react"
import renderer from "react-test-renderer"

import Layout from "../layout"

jest.mock("../header", () => "Header")

describe("Layout", () => {
    it("renders correctly", () => {
        const layout = renderer
            .create(
                <Layout>
                    <span>Child</span>
                </Layout>
            )
            .toJSON()
        expect(layout).toMatchSnapshot()
    })
})
