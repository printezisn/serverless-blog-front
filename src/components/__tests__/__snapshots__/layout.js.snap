// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<div>
  <Header />
  <main
    className="container"
  >
    <span>
      Child
    </span>
  </main>
</div>
`;
